timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -586,6 +612,77 @@ export default {
         this.meta.filter_labels.push(label)
       }
     },
+    toggleshowShortcutHelp: function () {
+      this.showShortcutHelp = !this.showShortcutHelp
+    },
+    nextpage(event) {
+      this.tableOptions.page++
```

Comment:
Since Johan moved the EventList into its own component, there is no `tableOptions` in this Explorer.vue component anymore. Hence this function throws an error.
I would recommend moving this feature to the EventList.vue component.